code/game/objects/items/weapons/twohanded.dm

```diff
@@ -323,13 +323,27 @@
 	item_state = "syn_breacher"
 	force_wielded = MELEE_FORCE_VERY_STRONG
 	really_heavy = TRUE
+	var/move_delay_addition = 1.5
 
 /obj/item/weapon/twohanded/breacher/synth/pickup(mob/user)
 	if(!(HAS_TRAIT(user, TRAIT_SUPER_STRONG)))
-		to_chat(user, SPAN_WARNING("You barely manage to lift \the [src] above your knees. This thing will probably be useless to you."))
+		to_chat(user, SPAN_HIGHDANGER("You barely manage to lift \the [src] above your knees. This thing will probably be useless to you."))
+		user.apply_effect(6, EYE_BLUR)
+		user.apply_effect(3, DAZE)
+		user.emote("pain")
```

Comment:
Emoting pain is a bit much, slow down is fine, make effects last less and no daze